### PR TITLE
[TW-1219] Apply template during workspace creation

### PR DESCRIPTION
**Pull request by MatthewPinsker-postman**
* Added that you can choose a workspace template when you create a new workspace
* Only available on Free, Basic, and Professional plans
* Updated relevant screenshots

### CODE REVIEW

**Review by mark-anthony-postman — APPROVED** [review]
LGTM!

File: src/pages/docs/collaborating-in-postman/using-workspaces/creating-workspaces.md [hunk 1]
```diff
@@ -54,17 +54,23 @@ _Workspace as an element_ represents a whole container where being an Admin give
 
     <img alt="Create new workspace" src="https://assets.postman.com/postman-docs/v10/workspace-switcher-v10.14.jpg" width="300px"/>
 
+1. ([Free, Basic, and Professional plans only](https://www.postman.com/pricing)) You can use [workspace templates](#creating-workspaces-with-a-template) to help you set up a new workspace. Select a workspace template to populate the workspace with helpful information and sample collections, or select **Blank workspace** to create an empty workspace. Then select **Next**.
+
+    > You can also apply a workspace template to a newly created workspace. To learn more, see [Creating workspaces with a template](#creating-workspaces-with-a-template).
+
+    <img alt="Create new workspace" src="https://assets.postman.com/postman-docs/v10/create-workspace-and-apply-template-v10.jpg" width="400px"/>
```
Comment (ramji367) [inline src/pages/docs/collaborating-in-postman/using-workspaces/creating-workspaces.md:61]: For this screenshot can we show the whole screen - the list of templates on the left along with the preview of the template on the right?

**Review by ramji367 — CHANGES_REQUESTED** [review]
Added one request to update a screenshot. Otherwise everything else looks good!

**Review by ramji367 — APPROVED** [review]
Looks good to me!

**Review by jkonrath-postman — APPROVED** [review]
LGTM

**Pull request merged**